steam/ext/csgo/client.py

```diff
@@ -58,6 +62,71 @@ async def _handle_ready(self) -> None:
         self.http.user = ClientUser(self._connection, await self.http.get_user(self.user.id64))
         await super()._handle_ready()
 
+    @overload
+    async def inspect_item(self, *, owner: Type[SteamID], asset_id: int, d: int) -> PreviewDataBlock:
+        ...
+
+    @overload
+    async def inspect_item(self, *, s: int, a: int, d: int) -> PreviewDataBlock:
+        ...
+
+    @overload
+    async def inspect_item(self, *, m: int, a: int, d: int) -> PreviewDataBlock:
+        ...
+
+    @overload
+    async def inspect_item(self, *, url: str) -> PreviewDataBlock:
+        ...
+
+    async def inspect_item(
+            self,
+            *,
+            owner: SteamID | None = None,
+            asset_id: int | None = None,
+            s: int = 0,
+            a: int = 0,
+            d: int = 0,
+            m: int = 0,
+            url: str = "",
+            timeout: int=10
+    ) -> PreviewDataBlock:
+        """
+        The parameters can be taken from `inspect` links either from an inventory or market.
+        The market has the `m` parameter, while the inventory one has `s`.
+        :param s: steam id64 of owner
```

Comment:
I'd rather have the user not pass this, s can just be removed, they should pass owner.